pandas/tests/indexes/test_base.py

```diff
@@ -759,6 +759,14 @@ def test_symmetric_difference(self):
         expected = MultiIndex.from_tuples([('bar', 2), ('baz', 3), ('bar', 3)])
         self.assertTrue(tm.equalContents(result, expected))
 
+        # Test for equal multiIndexes
```

Comment:
move all of the MultiIndex set-op tests to test_multi.py (including the new one).